### PR TITLE
Small fixes for PR #23 into upstream

### PR DESCRIPTION
**Pull request by ellenhp**
Just some tweaks that I had to make to get my smoke test working with podman-compose and get bind mounts working on SELinux distros (Fedora).

### CODE REVIEW

**Comment by ellenhp** [comment]
~~I don't know if `never` is right here for the restart policy but `no` didn't work with podman-compose, and I'd prefer to not break my workflow. If this breaks docker-compose maybe we can just leave the defaults?~~

~~```Error: running container create option: "False" is not a valid restart policy: invalid argument```~~

~~It's also possible I needed to add quotes around `no`. I'll check that now.~~

Figured it out!

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ cargo run --bin indexer -- \
 load-osm-pbf /data/australia-oceania-latest.osm.pbf
 
 # Run service
-cargo run --bin airmail_service -- \
```
Comment (tristan-morris) [inline README.md]: I'm surprised this works for you?  Is it worth making that the default bin or renaming accordingly?

```bash 
cargo run --bin airmail -- \
--index /data/index/
error: no bin target named `airmail`.
Available bin targets:
    indexer
    merge
    query
    airmail_service
```

**Comment by tristan-morris** [comment]
Perfect. Wow that sounds frustrating the no vs "no" with podman. What on earth is going on with the underlying type.

CLA bot shouldn't be on for this project.. :/ I'll fix that up.

Just added one comment

File: README.md (hunk 1 above)
Comment (ellenhp) [reply README.md]: Ooooooops, I misread the instructions apparently.

**Comment by ellenhp** [comment]
Fixed the mistake in the README. I did still have to make a change but I definitely made it in the wrong place first. 😅

**Comment by tristan-morris** [comment]
Haha all good. Just merged it now :)

**Pull request merged**